tutor/src/components/task-plan/builder/tasking-date-times.cjsx

```diff
@@ -27,12 +27,6 @@ TaskingDateTimes = React.createClass
   getDefaultProps: ->
     bsSizes: { sm: 8, md: 9 }
 
-  mixins: [BindStoreMixin]
-  bindStore: TaskingStore
-  bindEvent: ->
-    type = if @props.period then @props.period.id else 'all'
-    "taskings.#{@props.id}.#{type}.changed"
-
   getError: ->
```

Comment:
@pandafulmanda without this does the visible dates update on the mini-editor when you create from a cloned plan?

Reply:
yes

Reply:
we should chat about this quickly